substra/sdk/backends/remote/rest_client.py

```diff
@@ -212,7 +212,7 @@ def _add(self, name, exist_ok=False, **request_kwargs):
             if not exist_ok:
                 raise
 
-            key = e.pkhash or e.computePlanID
+            key = e.key
```

Comment:
Are you sure you can remove handling the computePlanID here? Are we sure there cannot be an "AlreadyExists" exception when creating a compute plan?

Reply:
See there : https://github.com/SubstraFoundation/substra/pull/220#discussion_r482822939